package.json

```diff
@@ -29,7 +29,7 @@
   },
   "rnpm": {
     "assets": [
-	      "./assets/fonts/"
+      "./assets/fonts/"
     ]
```

Comment:
We can take this out - it was for importing a custom font. However, we don't need it at the moment

Reply:
ok, removed